test_accounts/health_test_accounts.md

```diff
@@ -11,7 +11,9 @@ Use these test accounts to log into the ID.me sandbox environment that is connec
 ---
 
 **NOTE:**
-Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx)
+
+ * Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx).
```

Comment:
One other quick thing: I'd replace "to" with "with" --> "Details about Conditions associated with a patient can be found..."

Reply:
Updated

Reply:
Looks good to me!